### ISSUE TITLE
No tilesSpec needed

### ISSUE THREAD

**Issue by BHare1985** [landtechnologies/Mapbox-vector-tiles-basic-js-renderer#8]
I am a little confused by the concept of this project. It looks like you offer the ability to draw a canvas overlay on google maps that renderes based on vector tiles but it requires a "tilesSpec" to be defined on which tiles are included.

What I am after is the ability to render a google map overlay that is coming from pbf vector tiles, but not needing to specify any tileSpec, I want every tile to be in the spec.

So in the context of your openstreet map demo, every tile would get a OSM overlay, not requiring you to define specific tiles as the whole world is covered

Is this still a doable use-case for this project? if so how would I go about doing that.

**Comment by d1manson** [comment]
The main part of this project is the tilespec stuff, but there is a demo included, showing how to use that to make a Google map overlay, and indeed that is the main way we use it at landInsight. So if you can get the demo up and running  you dont need to worry about the underlying tilespec. 

Note that the demo code assumes you only have one zoom level for your tilset and renders that at all zoom levels. Also note that the underlying tilespec stuff is fairly basic when it comes to the placement of symbols and text.  

But aside from that this may do what you need.

**Comment by BHare1985** [comment]
Yeah I got the demo working but there is still a tilespec defined. If you zoom out you can see where the rendering just stops. If I have a URL that is the pbf of the whole world, how can I get it render for the whole world without having the need to define every single tile in the tilespec
